### ISSUE TITLE
Switch to Fig from `zsh-autosuggestions`

### ISSUE THREAD

**Issue by yeskunall** [yeskunall/dotfiles#16]
This one is a _maybe_, but [Fig](https://fig.io/) does look really cool 🤩

**Comment by yeskunall** [comment]
__They do not have an actual privacy policy yet__.

❌ Company HQ is in SF 🇺🇸

❌ Fig Dashboard (yeah, no thanks lol)
<details>
<img width="912" alt="image" src="https://user-images.githubusercontent.com/14703164/202090274-06cf8b6b-8c2f-4e68-be0d-eab4ec303d6d.png">
</details>

❌ Tracks autocomplete usage (why do you care which command I use, and how many times I use it?)
<details>
<img width="923" alt="image" src="https://user-images.githubusercontent.com/14703164/202091884-27ed1757-3713-40f0-8997-dac340ebc8dd.png">
</details>

❌ Tracks basic usage (why do you need to do this? Short answer: you don’t.)
<details>
<img width="913" alt="image" src="https://user-images.githubusercontent.com/14703164/202089711-4690f6f4-febd-4ef9-ad2e-c8ff73cd8439.png">
</details>

❌ Tracks terminal usage (fucking stop already)
<details>
<img width="928" alt="image" src="https://user-images.githubusercontent.com/14703164/202091623-4c0b2b71-0e47-4837-bef0-51e17c07e1e5.png">
</details>

❌ Uses Sentry for error reporting (this is set up by a human. Humans cannot be trusted. Next best option is to _verify_ they are only reporting on what they are saying, but their Sentry dashboard is not public)
<details>
<img width="937" alt="image" src="https://user-images.githubusercontent.com/14703164/202089926-7e22e9c2-7305-47f8-9087-8ee7b9a0e2fd.png">
</details>

❌ An email login to sync dotfiles, workflows, personal shortcuts SSH credentials, and more across devices (seriously? Bullshit excuse. They just want to be able to force you from _not_ using their product should they find you in violation of their ToS, which __this company fails to provide__.
<details>
<img width="915" alt="image" src="https://user-images.githubusercontent.com/14703164/202092476-31524da0-4097-41f1-ab73-aee73bcd0625.png">
</details>

They claim all telemetry can be disabled, but why would I trust them when they obssesively track what I’m doing in my terminal.

tl;dr: Security ≳ Privacy > Stability > Features
